### PR TITLE
Follow up to #6684 - Alignment issue for `Token grants claimed` in BR panel

### PR DESCRIPTION
**Pull request by masparrow**
Resolves https://github.com/brave/brave-browser/issues/7198

## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- Verified that these changes build without errors on
  - [ ] Android
  - [ ] iOS
  - [ ] Linux
  - [x] macOS
  - [ ] Windows
- Verified that these changes pass automated tests (unit, browser, security tests) on
  - [ ] iOS
  - [ ] Linux
  - [x] macOS
  - [ ] Windows
- [x] Verified that all lint errors/warnings are resolved (`npm run lint`)
- [x] Ran `git rebase master` (if needed).
- [ ] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request as needed.
- [ ] Request a security/privacy review as needed.
- [ ] Add appropriate QA labels (QA/Yes or QA/No) to include the closed issue in milestone
- [ ] Public documentation has been updated as necessary. For instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protection-Mode
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-compatibility-issues-with-tracking-protection
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide

## Test Plan:
As per https://github.com/brave/brave-browser/issues/7198

Although this is a test string, the line height has been tightened up as follows;

Screenshot 2019-12-06 at 10 01 10

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Request a security/privacy review as needed.
- [ ] Adequate test coverage exists to prevent regressions 
- [ ] Verify test plan is specified in PR before merging to source

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on.
- [ ] All relevant documentation has been updated.

### CODE REVIEW

**Comment by masparrow** [comment]
@NejcZdovc Updated with padding as per change request - using 10px top/bottom;

Screenshot 2019-12-05 at 10 09 19

File: components/brave_rewards/resources/ui/components/listToken/style.ts [hunk 1]
```diff
@@ -19,11 +19,12 @@ export const StyledWrapper = styled<Partial<Props>, 'div'>('div')`
 
 export const StyledTitle = styled<{}, 'div'>('div')`
   font-size: 14px;
-  line-height: 2.79;
+  line-height: 1.3;
   color: #4b4c5c;
   flex-grow: 1;
   flex-shrink: 1;
   flex-basis: 60%;
+  padding:10px 0px;
```
Comment (NejcZdovc) [inline components/brave_rewards/resources/ui/components/listToken/style.ts]: missing spacing after `:`

Comment (masparrow) [reply components/brave_rewards/resources/ui/components/listToken/style.ts]: Done

**Comment by NejcZdovc** [comment]
CI failed on `macos` (unrelated), but this change is only CSS so merging

**Pull request merged**